cmd/antrea-agent/agent.go

```diff
@@ -192,6 +193,14 @@ func run(o *Options) error {
 	}
 	go apiServer.Run(stopCh)
 
+	// Enable flow exporter
+	if o.config.EnableFlowExporter {
+		// Create connection store and builder. Run the builder.
+		connTrack := connections.NewConnTrackPoller(nodeConfig, connections.NewConnTrack())
+		cxnStoreAndBuilder := connections.NewCxnStoreAndBuilder(connTrack, ifaceStore)
```

Comment:
nit: why use both the `conn` and the `cxn` prefix? Can't we use `conn` everywhere or do they mean different things here?

Reply:
I tried to differentiate conntrack connection and antrea connection. I agree it looks awkward to have different names and may be confusing too. I just sticked with conn.